.github/CHANGELOG.md

```diff
@@ -26,6 +26,48 @@
 
 * Ising YY gate functionality added.
   [(#1358)](https://github.com/PennyLaneAI/pennylane/pull/1358)
+  
+* Added functionality to `qml.sample()` to extract samples from the basis states of
+the device (currently only for qubit devices). Additionally, `wires` can be
+specified to only return samples from those wires. 
+[(#1441)](https://github.com/PennyLaneAI/pennylane/pull/1441)
```

Comment:
```suggestion
* Added functionality to `qml.sample()` to extract samples from the basis states of
  the device (currently only for qubit devices). Additionally, `wires` can be
  specified to only return samples from those wires. 
  [(#1441)](https://github.com/PennyLaneAI/pennylane/pull/1441)
```

When writing lists in markdown, the continuation lines should match the indentation of the first line!